test/distributed/pipelining/model_registry.py

```diff
@@ -71,13 +68,15 @@ def forward(self, x):
 
 # Multi-MLP model
 class MultiMLP(torch.nn.Module):
-    def __init__(self, d_hid):
+    def __init__(self, d_hid: int, n_layers: int = 2):
         super().__init__()
-        self.mlp0 = MLPModule(d_hid)
-        self.mlp1 = MLPModule(d_hid)
+        self.layers = torch.nn.ModuleList([MLPModule(d_hid) for _ in range(n_layers)])
```

Comment:
I didn't check how you tested w/ manual frontend but if you make this a moduledict it may be easier to split this for manual stage.

Reply:
i see, nevermind.  becuase top level's forward is trivial, we can just get_submodule and that's all we need.